tap_github/teams.json

```diff
@@ -79,9 +79,12 @@
     "parent": {
       "type": [
         "null",
-        "object"
+        "string"
       ],
-      "properties": {}
+      "properties": [
```

Comment:
This looks like an error. I think this `"properties"` should be removed.

Reply:
good catch!!